lstmcpipe/stages/mc_process_dl1.py

```diff
@@ -173,8 +181,7 @@ def r0_to_dl1(
 
     log.info("Working on DL0 files in {}".format(input_dir))
 
-    check_data_path(input_dir)
-    raw_files_list = [file.resolve().as_posix() for file in Path(input_dir).rglob("**/*.simtel.gz")]
+    raw_files_list = get_input_filelist(rglob_pattern="*.simtel.gz")
```

Comment:
simtel extension might change in the near future, see:
https://github.com/cta-observatory/lst-sim-config/issues/42#issuecomment-1104898584

Shall we address that possibility already?